src/components/MoreJobs.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>{{ $t("More jobs") }}</ion-card-title>
+    </ion-card-header>
+    <ion-item v-for="job in moreJobs" :key="job" @click="viewJobConfiguration(job)" detail button>
```

Comment:
Key should be unique and should not be an object.

Reply:
Sure sir, I will use "jobId" as key.